programs/mango-v4/src/state/mango_account_components.rs

```diff
@@ -284,6 +284,10 @@ impl PerpPosition {
         self.base_position_lots
     }
 
+    pub fn effective_base_position_lots(&self) -> i64 {
+        self.base_position_lots + self.taker_base_lots
```

Comment:
this ignores what is in orders, because I enforce only one order at a time, wdyt?

Reply:
I don't see "only one order" being enforced. Can you add a comment saying that this takes into account base lots from unprocessed events, but not anything from open orders?

Reply:
I thought we agreed on not enforcing - only one order - when placing a new perp order on a perp market in reduce only mode, or?

Reply:
ok as discussed there would be no enforcing